src/main/323C.c

```diff
@@ -1576,23 +1621,61 @@ INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80022730);
 
 INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002328C);
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80023624);
+void func_80023624(struct EngineObj* arg0)
+{
+    reset_objects();
+    func_8002AB20();
+    arg0->unk1F = 0;
+    arg0->enable_boss = 0;
+
+    if (arg0->unk43 == 0) {
+        func_80018000(5);
+    } else {
+        func_80018000(10);
+    }
+
+    arg0->unk1++;
+}
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80023684);
+void func_80023684(struct EngineObj* arg0)
+{
+    arg0->unk1++;
+}
 
 INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80023698);
 
 INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_800237E4);
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80023870);
+void func_80023870(struct EngineObj* arg0)
+{
+    struct MiscObj* obj;
+
+    if (--arg0->unk4 == 0) {
+        arg0->unk1++;
+        obj = find_free_misc_obj();
+        if (obj != NULL) {
+            obj->base.active = 0x41;
+            obj->base.id = 0x38;
+            obj->base.unk2 = 2;
+            obj->ext.title_logo.palette_shift_speed = 0;
```

Comment:
Not sure if this is right

Reply:
It's not the same object as title_logo since id 0x38 should be func_80023CA4 I think? id is an offset into misc_object_update_funcs. It's possible the ext is wrong though, it's not a big deal to have this for the moment